### PR TITLE
Moved to SF 5.3 and PHP8

### PR DESCRIPTION
**Pull request by Tom32i**

### CODE REVIEW

**Review by benji07 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	vendor/bin/phpstan analyse .
+
+lint.composer:
+	composer validate --strict
+
+############
+# Security #
+############
+
+security.symfony@integration:
```
Comment (ogizanagi) [inline Makefile]: Not used? Actually I don't think it makes much sense for a library anyway.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+      - name: 'Cache composer dependencies'
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.directory }}
+          key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-composer-
+
+      - name: 'Fixup Composer'
+        if: matrix.php == 8.0
```
Comment (ogizanagi) [inline .github/workflows/ci.yaml]: Toujours utile ?

File: composer.json [hunk 3]
```diff
@@ -8,26 +8,32 @@
...
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/framework-bundle": "~3.0|~4.0|~5.0",
-        "symfony/form": "~3.0|~4.0|~5.0",
-        "ext-json": "*"
+        "php": "^7.4|^8.0",
+        "ext-json": "*",
+        "symfony/framework-bundle": "~4.0|~5.0",
```
Comment (ogizanagi) [inline composer.json]: Bump 4.4 LTS pour toutes les deps symfony ?

**Review by ogizanagi — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Tom32i) [reply .github/workflows/ci.yaml]: Je vais faire le test sans

**Pull request merged**